apps/judicial-system/web/src/components/Table/Table.tsx

```diff
@@ -123,14 +123,16 @@ const Table: React.FC<TableProps> = (props) => {
                   }`}
                 </Text>
               )}
-            {theCase.courtDate && (
-              <Text fontWeight="medium" variant="small">
-                {`${formatMessage(strings.hearing)} ${formatDate(
-                  parseISO(theCase.courtDate),
-                  'd.M.y',
-                )} kl. ${formatDate(parseISO(theCase.courtDate), 'kk:mm')}`}
-              </Text>
-            )}
+            {theCase.postponedIndefinitelyExplanation
```

Comment:
I feel like we're heading back into dangerous territory with having such specific logic inside the new Table component that we wanted to make more generalized, but I made it in to a ticket as I'd say it's a separate issue and more of a cooldown task than anything connected to this specific implementation.

Reply:
Sounds good, I didn't know how could be done differently.